File: Selenium/Activities/Activity19.java
package activity;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity19 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/alerts");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		WebElement confirmBtn = driver.findElement(By.id("confirmation"));
		confirmBtn.click();
		
		Alert alert = driver.switchTo().alert();
		
		System.out.println("Text of the alert: " + alert.getText());
		
		alert.accept();
		
		confirmBtn.click();
		
		alert.dismiss();
		
		driver.quit();
		
	}

}

File: Selenium/Activities/activity_19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, timeout=5)

    driver.get("https://training-support.net/webelements/alerts")

    print("Title of the page: ", driver.title)

    confirm_btn = driver.find_element(By.ID, "confirmation")
    confirm_btn.click()

    alert = wait.until(EC.alert_is_present())
    print("Text of the Alert: ", alert.text)

    alert.accept()

    confirm_btn.click()

    alert.dismiss()
    